web/src/engine/osk/src/views/oskView.ts

```diff
@@ -254,22 +253,10 @@ export default abstract class OSKView
       this.uiStyleSheetManager.linkExternalSheet(sheetHref);
     }
 
-    this.activeKeyboard = this.config.keyboardToActivate;
-
-    // Ensure the appropriate banner mode is activated; it's possible for a model to have either
-    // partially or fully loaded, especially if the prior line has a full Keyboard instance.
-    const modelState = this.config.predictionContextManager?.modelState;
-    if(modelState) {
-      this.bannerController.selectBanner(modelState);
-    }
-
     this.setBaseMouseEventListeners();
     if(this.hostDevice.touchable) {
       this.setBaseTouchEventListeners();
     }
-
-    this._Box.style.display = 'none';
```

Comment:
This line was not added in #11174; it was added in #11258. So, this PR should note: 

Reverts: #11258

Reply:
Good catch, but in the other direction; I believe this line should not be reverted, then.  Keeping `display: none` active on initialization helps prevent layout reflow / thrashing while we want on the "true" first keyboard to be set for loading.

Reply:
I'd want to see that come in as a separate PR with its own testing. My purpose here is to revert to a known-good state, not a presumably-good state.

Reply:
I'd like to point out that when I built the reversion, I did have to resolve a few merge conflicts.  I'd have to double-check, but there's a decent chance that this may have been one of the spots I had to resolve.

As you've pointed out, that line's removal is from a different PR than was intended for reversion - one which _precedes_ the reversion target in history.  These facts make me suspect that deleting this line would quality for the "presumably-good state" instead.

Reply:
Had a comment here I just deleted; got my history order wrong; please ignore any email notifications about that.

Reply:
Yay for an incoming reversion-reversion in the near future, then.

Reply:
Probably, that's life. But given that PR built on this one and seems linked to the change, I still would like to err on the side of caution, particularly because we are back-porting to 17.0.

Really happy to see it reintroduced in 18.0 alpha and go through test.